### PR TITLE
add mongoose and design mongoose schema

### PR DESCRIPTION
**Pull request by NDmorsalin**
I have added mongoose and connect MongoDB atlas URL and design todo schema

### CODE REVIEW

**Review by ArmanSarker — COMMENTED** [review]
Call api on  inputField   component and send a post reaquest to this api

File: src/Model/todoModal/todoModel.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* import { Schema, Document, model, Model } from 'mongoose';
+
```
Comment (TanvirAlam) [inline src/Model/todoModal/todoModel.ts:2]: Never keep commented out code ever!

File: src/Model/todoModal/todoModel.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+export default Todo;
+ */
+
+import { client } from "@/db/db.config";
+
+const database = client.db("feedhelp");
+const Todo = database.collection('todo')
+
+export default Todo
```
Comment (TanvirAlam) [inline src/Model/todoModal/todoModel.ts:42]: Always make sure you new line

File: src/app/Provider/QuaryProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    return (
+         <QueryClientProvider client={queryClient}>
+            {
+                children
+            }
+        </QueryClientProvider>
+    );
+};
+
+export default QueryProvider;
```
Comment (TanvirAlam) [inline src/app/Provider/QuaryProvider.tsx:25]: same here

File: src/app/api/todo/incomplete/route.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        
+    }
+
+   return  NextResponse.json({
+        status: 200,
+        body: {
+            message: 'getting todos'
+        }
+    })
+}
```
Comment (TanvirAlam) [inline src/app/api/todo/incomplete/route.ts:26]: same

**Pull request merged**